libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,23 @@
+let scopeUtils = require('taskcluster-lib-scopes');
```

Comment:
I suspect this is why netlify is failing: it's a circular requirement.  Instead:
```suggestion
const {normalizeScopeSet, scopeCompare} = require('./normalize');

```

Reply:
Thanks, I think that should fit it!